Bookish/Controllers/MemberController.cs

```diff
@@ -36,8 +44,15 @@ public ActionResult EditMember(int id)
     [HttpPost]
     public ActionResult EditMember(MemberModel replaceMember)
     {
-        MemberEditService.AlterMember(replaceMember.Id, replaceMember);
-        return RedirectToAction(nameof(MemberList));
+        if (MemberSearchService.IsFormBlank(replaceMember))
+        {
+            return View(replaceMember);
+        }
+        else
+        {
+            MemberEditService.AlterMember(replaceMember.Id, replaceMember);
+            return RedirectToAction(nameof(MemberList));
+        }

```

Comment:
But most of the properties on Member aren't nullable, so inserting will fail if they're not provided.

Reply:
I hadn't even noticed that - I meant to set them as nullable but must have accidentally forgotten, and then only tested the Book version and not the Member version.